plugin/megaraid/megaraid.py

```diff
@@ -304,6 +406,13 @@ def capabilities(self, system, flags=Client.FLAG_RSVD):
         cap.set(Capabilities.VOLUME_RAID_INFO)
         cap.set(Capabilities.POOL_MEMBER_INFO)
         cap.set(Capabilities.VOLUME_RAID_CREATE)
+        cap.set(Capabilities.BATTERIES)
+        cap.set(Capabilities.VOLUME_CACHE_INFO)
+        cap.set(Capabilities.VOLUME_PHYSICAL_DISK_CACHE_SET)
+        cap.set(Capabilities.VOLUME_WRITE_CACHE_POLICY_SET_WRITE_BACK)
+        cap.set(Capabilities.VOLUME_WRITE_CACHE_POLICY_SET_AUTO)
+        cap.set(Capabilities.VOLUME_WRITE_CACHE_POLICY_SET_WRITE_THROUGH)
+        cap.set(Capabilities.VOLUME_READ_CACHE_POLICY_SET)
```

Comment:
Why advertise this feature `VOLUME_READ_CACHE_POLICY_SET` when the plugin will return `NO_SUPPORT`?